test/common/alerts_firing.go

```diff
@@ -26,11 +32,20 @@ type alertsFiringError struct {
 	alertsPending        []alertTestMetadata
 	alertsFiring         []alertTestMetadata
 	deadMansSwitchFiring bool
+	MasterURL            string
 }
 
+var (
+	podNamespaces = []string{
+
+		"redhat-rhmi-3scale", "redhat-rhmi-3scale-operator", "redhat-rhmi-amq-online", "redhat-rhmi-apicurito", "redhat-rhmi-apicurito-operator", "redhat-rhmi-cloud-resources-operator", "redhat-rhmi-codeready-workspaces", "redhat-rhmi-codeready-workspaces-operator", "redhat-rhmi-fuse", "redhat-rhmi-fuse-operator", "redhat-rhmi-middleware-monitoring-operator", "redhat-rhmi-middleware-monitoring-federate", "redhat-rhmi-operator", "redhat-rhmi-rhsso", "redhat-rhmi-rhsso-operator", "redhat-rhmi-solution-explorer", "redhat-rhmi-solution-explorer-operator", "redhat-rhmi-ups", "redhat-rhmi-ups-operator", "redhat-rhmi-user-sso-operator", "redhat-rhmi-user-sso",
```

Comment:
Are these all of our product namespaces? I know we have a list of product and operator namespaces defined in the `common/types.go` - https://github.com/integr8ly/integreatly-operator/blob/master/test/common/types.go#L18-L39. Any additional ones can just be added on to your list. 
I think using those variable names instead and avoiding duplication will make this a lot easier to maintain in the future and a lot less error prone! 
Also could make sense to break these into single lines for better readability

Reply:
Ohh handy out i didnt know about these ill change them over